PowerEditor/installer/nativeLang/swedish.xml

```diff
@@ -305,7 +306,8 @@
 					<Item id="50006" name="Sökvägskomplettering"/>
 				</Commands>
 			</Main>
-			<Splitter />
+            <Splitter>
+            </Splitter>
```

Comment:
@donho I know that you don't like to make formatering changers, and I absolutely understand the motivation. I agree that it is not good to mix regular changes with formatering changes, as it is hard to see the changes that matter. 

The problem is that the quality of the code slowly deteriorates if it is not allowed to improve the formatting.

I agree that it is not wise to make big changes at once (which I stupidly tried to do), but it must be allowed to make small improvements in order to at least improve the quality.

I did not think it was so controversial to at least fix a few simple line feeds, but I restore them now.

The reason I want to do this is because it is very difficult to compare the Swedish translation with the English source. It had make it much simpler if I was allowed to do the translation equivalent to the English original.